tests/datasource/test_pandas_datasource.py

```diff
@@ -142,7 +142,8 @@ def test_pandas_datasource_custom_data_asset(data_context, test_folder_connectio
 
 def test_pandas_source_read_csv(data_context, tmp_path_factory):
     if not PY3:
-        pytest.skip(reason="We don't specifically test py2 unicode reading since this test is about our handling of kwargs *to* read_csv")
+        pytest.skip("We don't specifically test py2 unicode reading since this test is about our handling of kwargs *to* read_csv")
+
```

Comment:
Nitpick - I know this was existing code - I find this sort of thing much more readable as:
```
@pytest.mark.skip(condition=PY2, reason="We don't specifically test py2 unicode reading since this test is about our handling of kwargs *to* read_csv")
def test_pandas_source_read_csv(data_context, tmp_path_factory):
...
```

Reply:
:+1:

TIL

Reply:
Tagging @jcampbell , since it looks like he wrote the line originally. If we're going to settle on pytest.mark as a convention, he should know it too.